config/scenario_config.csv

```diff
@@ -2,55 +2,47 @@ title;start;CES_parameters;optimization;slurmConfig;regionmapping;extramappings_
 SSP2EU-PkBudg650;1,AMT;;;;;;rcp20;9;globallyOptimal;650;diffCurvPhaseIn2Lin;100;2080;;9;;;;;;;;;;;;NDC;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;on;on;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;Mix4;2025;;SSP2EU-NPi;SSP2EU-NPi;;SSP2EU-PkBudg650: This climate policy scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The stylized climate policy scenario assumes a peak budget of 650 Gt?CO2 on total CO2 emissions from 2015 to 2100. This is a 1.5??C scenario, peak warming is allowed to be at or slightly above 1.5??C, at median climate sensitivity but returns to values below 1.5??C in at least 67?% of scenarios by the end of the century.
 SSP2EU-PkBudg1050;1,AMT;;;;;;rcp26;9;globallyOptimal;1050;diffCurvPhaseIn2Lin;60;2100;;9;;;;;;;;;;;;NDC;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;on;on;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;Mix3;2025;;SSP2EU-NPi;SSP2EU-NPi;;SSP2EU-PkBudg1050: This climate policy scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The stylized climate policy scenario assumes a peak budget of 1150 Gt?CO2 on total CO2 emissions from 2015 to 2100. This is a well below 2??C scenario, at median climate sensitivity but returns to values below 2??C in at least 67?% of scenarios during the whole century.
 # EU21 SSP2EU;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-SSP2EU-EU21-NPi-calibrate;calibrate;calibrate;;15;./config/regionmapping_21_EU11.csv;./config/extramapping_EU27.csv;rcp45;;;0;NPi;1;2100;;9;;;;;;;;;;;;NPi2018;regiCarbonPrice;;;on;on;EUR_regi, NEU_regi;;;;;;;;;;3;;;;;;Mix1;2005;;;;;SSP2EU-EU21-calibration: This baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
-SSP2EU-EU21-Base;1,AMT;;;;./config/regionmapping_21_EU11.csv;./config/extramapping_EU27.csv;;;;0;;;2100;;;;off;;;;;;;;;;;;;;;;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;Mix1;2005;;;;;SSP2EU-EU21-Base: This baseline calibration scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
+SSP2EU-EU21-NPi-calibrate;calibrate;calibrate;;14;./config/regionmapping_21_EU11.csv;./config/extramapping_EU27.csv;rcp45;;;0;NPi;1;2100;;9;;;;;;;;;;;;NPi2018;regiCarbonPrice;;;on;on;EUR_regi, NEU_regi;;;;;;;;;;3;;;;;;Mix1;2005;;;;;SSP2EU-EU21-calibration: This baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
```

Comment:
Why run `SSP2EU-EU21-NPi-calibrate` on twelve instead of 16 cores?

Reply:
I did a presentation in a remind meeting in 2019, when we merged REMIND-EU, that showed that increasing the number of cores from 12 to 16 had no gains for the total model solution time. 

At the time, the determinant for the time taken at each iteration was the solution time of the "most complex" regions, regions that had substantial amounts of primary energy exports or demand. The number of cores had not much influence because other regions solved much quicker in sequence as the other cores became available.

Unless someone did the work of evaluating this again and found a distinct behavior, changing to use 16 cores can be simply a waste of resources.

Reply:
Might be worth looking into that again with #1522.
I would say postpone this, to see if it is still true.  And if it is, change `scripts/start/choose_slurmConfig.R`, too, as it is misleading REMIND users.

Reply:
Reverted for now as suggested by the reviewer.